client/src/plugins/update-checks/UpdateChecks.js

```diff
@@ -363,7 +363,18 @@ export default class UpdateChecks extends PureComponent {
       _getGlobal
     } = this.props;
 
-    _getGlobal('backend').send('external:open-url', { url: latestVersionInfo.downloadURL });
+    let downloadURL = latestVersionInfo.downloadURL;
+
+    // Check if UTM tags are already present
```

Comment:
I wonder if this should not be a general utility that we can use across different places, i.e. 

```
utmTag(url, {
  campaign?: string
})
```